src/text-area/text-area.stories.ts

```diff
@@ -0,0 +1,42 @@
+import { html, TemplateResult } from "lit";
+import "./index.ts";
+
+export default {
+  title: "TextArea",
+  component: "tap-text-area",
+  argTypes: {
+    label: {
+      control: 'text',
+      description: 'label',
+    }
```

Comment:
Please add story controls for these props too:

- `caption`
- `placeholder`
- `disabled`
- `error`
- `value`